webpack/utils/env.js

```diff
@@ -0,0 +1,44 @@
+const chalk = require("chalk")
+const path = require("path")
+const yn = require("yn")
+
+const env = {
+  buildClient: yn(process.env.BUILD_CLIENT, { default: false }),
```

Comment:
Is `yn` only being used for `default`? Perhaps the `??` operator would suffice?

Reply:
`yn` smartly determines if a string is true or false. I want to avoid situations where we rely on presence of an environment variable to determine truthiness. `MY_ENV_VAR=false` should not be interpreted as true.

The default is unrelated to the usage of this library, it was a supported feature so I used it. 🤷